### DISCUSSION TITLE
Secret variables for request not getting version control

### DISCUSSION THREAD

**Discussion by LuccaRomanelli** [usebruno/bruno#1237] [Ideas]
In the enviroments we have an option to have Secret values like
![image](https://github.com/usebruno/bruno/assets/48338875/596df7bd-a380-438d-8d31-5030012dc3d3)

This is really usefull for tokens that for evrery creation you get one diferent.

I was thinking that a good implemetention for this would be in vars of request like: 
![image](https://github.com/usebruno/bruno/assets/48338875/ac28a983-bc65-4f44-90d6-12b273910417)
If I could get this variables to be secret I could have values for this and when I change in muy local it does not get version control, leaving free to change the contens of body and query params witchout the need to version control it

**Comment by helloanoop** [comment]
Closing as it is being tracked in https://github.com/usebruno/bruno/issues/5039
